runtime/runtime/src/congestion_control.rs

```diff
@@ -82,7 +89,12 @@ impl<'a> ReceiptSink<'a> {
             debug_assert!(ProtocolFeature::CongestionControl.enabled(protocol_version));
             let outgoing_buffers = ShardsOutgoingReceiptBuffer::load(trie)?;
 
-            let outgoing_limit: HashMap<ShardId, Gas> = apply_state
+            // One of the shards is designated as "special". We are allowed to send more receipts to this one "special" shard.
+            let num_shards = apply_state.congestion_info.len();
+            let special_shard: Option<ShardId> =
+                choose_special_shard(apply_state.shard_id, apply_state.block_height, num_shards);
```

Comment:
Maybe "high_flow"? Because there'll be a lot of receipts flowing there.
I don't like thinking too much about names because it's hard to find a really good name and feels a bit like bikeshedding.

Reply:
Agreed, naming is hard and can quickly derail a discussion into bikeshedding. And yet, in my opinion, bad names are one of the highest contributors to unreadable and unmaintainable code. In this case, it's not quite as bad so I suggested it as an optional change to reconsider.

I like "high_flow" better than "special".  I think it can also be a good name if we combine "allowed" from congestion control and this shard into the same. But your opinion is as valid as mine.

And again, renaming this is optional and not something to block on.

Reply:
> I think it can also be a good name if we combine "allowed" from congestion control and this shard into the same. 

Yeah, merging with `allowed` should fix the problem :)